server/controllers/finance/reports/cashflow/index.js

```diff
@@ -117,7 +117,7 @@ function reportByService(req, res, next) {
   db.one(cashboxDetailsSql, cashboxId)
```

Comment:
!!!  Can we rename this variable from `cashboxId` to `cashboxAccountId`?  The origin of this bug seems to be from a bad name.